website/screens/components/status-light/usage/StatusLightUsagePage.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  DxcParagraph,
+  DxcBulletedList,
+  DxcFlex,
+  DxcTable,
+} from "@dxc-technology/halstack-react";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import DocFooter from "@/common/DocFooter";
+import Example from "@/common/example/Example";
+import HeaderDescriptionCell from "@/common/HeaderDescriptionCell";
+import variants from "./example/variants";
+
+const sections = [
+  {
+    title: "Usage",
+    content: (
+      <DxcParagraph>
+        The Status light component serves as a powerful communicator of an
+        element or process' status. It focuses exclusively on conveying semantic
+        information within a range of five variants, each of them aligned with
+        its specific process status.
+      </DxcParagraph>
+    ),
+    subSections: [
+      {
+        title: "Do's",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              Match each status light variant with the correct semantic meaning
+              to communicate clearly and consistently messages about our
+              statuses.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Use each size of the component according to the different screen
+              sizes where it will be placed, paying special attention to
+              maintaining legibility and functionality.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Use concise and precise labels that reflect the status of a
+              process accordingly. You should not use more than three words in a
+              status light label.
+            </DxcBulletedList.Item>
+          </DxcBulletedList>
+        ),
+      },
+      {
+        title: "Don'ts",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              Mix semantic meanings within Status lights, as this will lead to
+              confusion and misinterpretation of process states.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Use Status light and semantic badges in the same data
+              visualization format (tables, charts, data grids...).
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Overuse the component. In screens with high cognitive load, you
+              should not overuse this component, as it may confuse users and
+              interrupt the reading flow.
+            </DxcBulletedList.Item>
+          </DxcBulletedList>
+        ),
+      },
+    ],
+  },
+  {
+    title: "Variants",
+    content: (
+      <>
+        <DxcParagraph>
+          The Status light component has five different modes, each one
+          corresponding to its semantic meaning:
```

Comment:
This sentence is a bit odd. Having the two dots and the line jump to again have two dots. I see two possibilities here:

1. Remove the first two dots and end the sentence with a single dot.
2. Join both sentences: "The Status light component has five different modes, each one corresponding to its semantic meaning: **default**, **info**, **success**, **warning** and **error**."

Reply:
You are right, I added the first option in order to follow the same structure as the `Button` Usage.